src/lib/utilities/API/imgurApi.ts

```diff
@@ -22,3 +22,20 @@ export const postImage = (
     ...config,
   });
 };
+
+// show the remaining limits to make requests to the imgurAPI
+// By defalut imgur provide us approximately 1,250 uploads per day or approximately 12,500 requests per day.
+// Max 12,500 requests - allowed during 1 month
+// If the daily limit is hit five times in a month, then the app will be blocked for the rest of the month
+export const getLimits = (
+  config = {
+    headers: {
+      Authorization: `Client-ID ${CLIENT_ID}`,
+    },
+  },
+): Promise<AxiosResponse> => {
+  return axios.get(`${BASE_URL}/credits`, {
```

Comment:
use axions instance provided above, you don't need to put headers explicetly here, there're intercetpors added

Reply:
fixed